graph_algorithms/README.algorithms

```diff
@@ -63,23 +63,26 @@ https://docs.tigergraph.com/graph-algorithm-library
 
 List of GSQL Graph Algorithms 
 -----------------------------
-as of December 14, 2018
+as of March 14, 2019
 Compatible with TigerGraph version 2.1.8 or higher
 
 closeness_cent                  Closeness Centrality
 conn_comp                       Connected Component Detection
 label_prop                      Label Propagation Method for Community Detection
-louvain                         Louvain Modularity Method for Community Detection
+louvain                         Parallel Louvain Modularity Method with Refinement for Community Detection
 pageRank                        PageRank measurement of relative influence of each vertex
+pageRank_wt                     Weighted PageRank
+pageRank_pers                   Personalized PageRank
 shortest_ss_no_wt               Single-Source Shortest Paths without weight
 shortest_ss_pos_wt              Single-Source Shortest Paths with positive weight
 shortest_ss_any_wt              Single-Source Shortest Paths
+Minimal Spanning Tree (MST)
```

Comment:
First column should be the query's name: mst
Second column is short descriptiobn